README.md

```diff
@@ -17,96 +12,42 @@ Clone the source code:
+### Frontend and Backend Installation
 
-```
-npm run db:reset
-```
+Refer to the READMEs for both the frontend and backend of the Measure Repository Service in their respective directories for specific setup and installation instructions.
 
 ## Usage
 
-Once MongoDB is running on your machine, run the `npm start` command to start up the Measure Repository Service server at `localhost:3000`.
-
-You can also run `npm run start:clean`, which will reset the database before starting up the Measure Repository Service server.
-
-For ease of testing, it is recommended to download [Insomnia API Client and Design Tool](https://insomnia.rest) for sending HTTP requests to the server and [Robo 3T](https://robomongo.org) as a GUI for viewing the Mongo database.
-
-When sending requests, ensure that the `"Content-type": "application/json+fhir"` header is set.
-
-### CRUD Operations
+Once you have the necessary dependencies installed in the `frontend` and `backend` directories, you can run the following in the root directory:
 
-This server currently supports the following CRUD operations:
+To start the the frontend application and backend server in parallel:
 
-- Read by ID to endpoint: `4_0_1/<resourceType>/<resourceId>`
-  _More functionality coming soon!_
-
-### Search
-
-The Measure Repository Service server supports `Library` and `Measure` resource search by the parameters specified in the shareable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#shareable-measure-repository). These parameters are:
-
-- description
-- identifier
-- name
-- status
-- title
-- url
-- version (can appear only in combination with a url search)
-
-### Package
-
-The Measure Repository Service server supports the `Library` and `Measure` `$package` operation with the `id` and `identifier` parameters and SHALL parameters specified in the shareable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#publishable-measure-repository). Accepted parameters are:
-
-- id
-- url
-- identifier
-- version
-
-### Data Requirements
-
-The Measure Repository Service server supports the `Measure` and `Library` `$data-requirements` operations with the SHALL parameters specified in the publishable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#publishable-measure-repository). SHALL parameters are:
-
-- id
-- url
-- version
-- identifier
+```bash
+npm run start:all
+```
 
-Supported optional parameters for `Measure` are:
+This starts up the Measure Repository service at `localhost:3000` and you can open the Measure Repository Service frontend application by navigating to [http://localhost:3001](http://localhost:3001) in your browser.
```

Comment:
```suggestion
This starts up the Measure Repository service at `localhost:3000` and you can open the Measure Repository Service frontend application by navigating to http://localhost:3001 in your browser.
```

markdown tip: you don't need to use the `[display](link)` syntax for a link that starts with http already if you don't want to display the actual linka s something different